pkg/services/folder/folderimpl/folder_test.go

```diff
@@ -732,6 +732,70 @@ func TestNestedFolderServiceFeatureToggle(t *testing.T) {
 	})
 }
 
+func TestFolderServiceDualWrite(t *testing.T) {
+	g := guardian.New
+	guardian.MockDashboardGuardian(&guardian.FakeDashboardGuardian{CanSaveValue: true})
+	t.Cleanup(func() {
+		guardian.New = g
+	})
+
+	db := sqlstore.InitTestDB(t)
+	cfg := setting.NewCfg()
+	features := featuremgmt.WithFeatures()
```

Comment:
[question] [non-blocker] Do we need `ProvideStore` to receive an empty `*FeatureManagement` or could we create `features` like this and reuse it when creating `Service`? 
```suggestion
	features := featuremgmt.WithFeatures(featuremgmt.FlagNestedFolders)
```

Reply:
actually `features` is redundant (is not used inside the nested folder store) so I have removed it
but to answer your question: yes, we can create the features like this and pass the same to the service.